float64               heading
float64               amrRoll
float64               apWndDir
float64               apWndSpd
float64               cog
float64               sog
float64               truWndSpd
float64               truWndDir
float64               lat
float64               long
